### DISCUSSION TITLE
Passkey stored in third party password manager

### DISCUSSION THREAD

**Discussion by Molenaar2** [ente-io/ente#3775] [Enhancements]
I could not find any relevant information when searching, but is it possible to have the Ente passkey created in third-party password managers. It looks I only can create device bound passkeys in Ente Auth. Would appreciate if the passkey could be stored in Bitwarden.

**Comment by dtemme** [comment]
I also got problems when trying to login on Android using Bitwarden as my passkey manager. I have to send the URL from Brave to my laptop to validate the passkey. Other website using passkeys are working.

For now I disabled passkeys and enabled TOTP as a seconds auth factor.
